docs/source/models/gbd2017/risks/low_birthweight_short_gestation/index.rst

```diff
@@ -0,0 +1,44 @@
+.. _2017_risk_lbwsg:
+
+====================================
+Low Birth Weight and Short Gestation
+====================================
+
+Risk Description
+-------------------
+
+Case definition in GBD 2017
++++++++++++++++++++++++++++
+
+The meaning of the “low birth weight” and “short gestation” in GBD have subtle
+definitional differences compared to other usages of “low birth weight” and
+“short gestation” in literature. The term “low birth weight” has historically
+been used to refer to birth weight (BW) less than 2500 grams. However, because
+the goal of the GBD risk factors analysis is to quantify the entirety of
+attributable burden due to each risk factor, the GBD definition of “low birth
+weight” therefore refers to all birth weight below the Theoretical Minimum Risk
+Exposure Level (TMREL) for birth weight. Likewise, new-borns have been typically
+been classified into gestational age (GA) categories of “extremely preterm” (<28
+weeks of gestation), “very preterm” (28-<32 weeks of gestation), and “moderate
+to late preterm” (32-<37 weeks of gestation). “Short gestation” in GBD refers to
+all gestational ages below the gestational age TMREL.
+
+Exposures and relative risks for the GBD Low birth weight and short gestation
+risk factors are divided into joint 500-gram birth weight and 2-week gestational
```

Comment:
For actual breakdown see:
[Doc - Apr 12 2019 - 9-04 AM.pdf](https://github.com/ihmeuw/vivarium_research/files/3985988/Doc.-.Apr.12.2019.-.9-04.AM.pdf)

Reply:
@collijk Should I include a link to that image in the documentation?